ci-operator/config/openshift/splunk-forwarder-images/openshift-splunk-forwarder-images-master.yaml

```diff
@@ -9,7 +9,7 @@ build_root:
     namespace: openshift
     tag: golang-1.15
 images:
-- dockerfile_path: containers/forwarder/Dockerfile
+- dockerfile_path: build/Dockerfile.forwarder
```

Comment:
We should just use `build/Dockerfile` like other single-image projects, [example](https://github.com/openshift/release/blob/efc48473bf53a8db53313430a23d7a025fdd6f56/ci-operator/config/openshift/configure-alertmanager-operator/openshift-configure-alertmanager-operator-master.yaml#L4)